build.gradle

```diff
@@ -6,25 +6,33 @@ repositories {
 defaultTasks 'build'
 
 configurations {
+	dbDriver { transitive = false }
 	tomcatLib { transitive = false }
 	cloverLib { transitive = false }
 	prometheus { transitive = false }
 }
 
 dependencies {
 	cloverLib "javax.jms:javax.jms-api:2.0.1"
-	
-	tomcatLib "com.microsoft.sqlserver:sqljdbc4:4.0"
-	tomcatLib "mysql:mysql-connector-java:5.1.47"
-	tomcatLib "com.oracle:ojdbc6:11.2.0.3"
-	tomcatLib "org.postgresql:postgresql:42.2.12"
+
+	dbDriver "com.microsoft.sqlserver:sqljdbc4:4.0"
+	dbDriver "mysql:mysql-connector-java:5.1.47"
+	dbDriver "com.oracle:ojdbc6:11.2.0.3"
+	dbDriver "org.postgresql:postgresql:42.2.12"
+
+	tomcatLib "org.bouncycastle:bcprov-jdk15on:latest.release"

```

Comment:
The prior instructions had the user manually downloading the latest version from the Bouncy Castle website, so I replicated that logic here. You may prefer to have this pinned to a specific version. I put remarks in the README to check for security updates, if they do pin the version.